docs/running_grafeas.md

```diff
@@ -156,4 +156,13 @@ func main() {
 		log.Println("Project does not contain any notes")
 	}
 }
-```
\ No newline at end of file
+```
+
+## Enable CORS on the sample server.
+
+### Update config
+
+Add the following to your config file below the `api` key.
+    cors_allowed_origins:
```

Comment:
Use a fenced code block to keep consistent with the rest of the document https://help.github.com/articles/creating-and-highlighting-code-blocks/

Reply:
I was trying to keep it consistent with the other [config example](https://github.com/grafeas/grafeas/blame/master/docs/running_grafeas.md#L83-L87) but didn't add the newline there. I'll add the newline to make it consistent with that other example.

Reply:
👍 sounds good!